docs/src/modules/components/AppNavDrawerItem.js

```diff
@@ -253,6 +253,7 @@ export default function AppNavDrawerItem(props) {
             activeClassName="app-drawer-active"
             href={href}
             underline="none"
+            noLinkStyle
```

Comment:
Remove the following React elements:

<img width="215" alt="Screenshot 2022-03-11 at 16 33 03" src="https://user-images.githubusercontent.com/3165635/157902016-36a27fc5-cca5-4ecd-b0cb-06f0c070fa96.png">